signature.gemspec

```diff
@@ -12,6 +12,9 @@ Gem::Specification.new do |s|
   s.summary     = %q{Simple key/secret based authentication for apis}
   s.description = %q{Simple key/secret based authentication for apis}
 
+
+  s.metadata['allowed_push_host'] = "https://rubygems.pkg.github.com/YourAcclaim"
```

Comment:
Curious why the same check isn't being done here as the other repos like: https://github.com/YourAcclaim/s3_image_processor/pull/1/files#diff-7aa6a00cdd54637ce521768ec7bded9f062fb2c935783d81821bb80c2b6f4c64R18-R22

Reply:
The main reason is that this repo is public, whereas the others are not. So the impact to our organization if this were accidentally pushed to a public registry is lower. (There's also an already-published version of this gem in RubyGems, which we don't have keys to push to, which makes it very unlikely we'd accidentally push it to the wrong host.)

Reply:
Ah I see. Thanks for pointing that out.